lib/gui/app/pages/main/templates/main.tpl.html

```diff
@@ -3,7 +3,7 @@
     <div class="box text-center relative" os-dropzone="image.selectImageByPath($file)">
 
       <div class="center-block">
-        <svg-icon path="main.selection.getImageLogo() || '../../assets/image.svg'"></svg-icon>
+        <svg-icon contents="main.selection.getImageLogo()" paths="[ '../../assets/image.svg' ]"></svg-icon>
```

Comment:
Isn't `contents` supposed to be an array?! See e.g. https://github.com/resin-io/etcher/pull/1677/files#diff-0bee66f458b50bfd24c37224e5891e78R105